transforms/code/license_check/ray/Makefile

```diff
@@ -0,0 +1,56 @@
+
+# Define the root of the local git clone for the common rules to be able
+# know where they are running from.
+REPOROOT=../../../..
+include $(REPOROOT)/transforms/.make.transforms
+
+TRANSFORM_NAME=license_check
+# $(REPOROOT)/.make.versions file contains the versions
+DOCKER_IMAGE_VERSION=${LICENSE_CHECK_RAY_VERSION}
+
+# Use default rule inherited from makefile.common
+clean:: .transforms.clean
+
+# Use default rule inherited from makefile.common
+test:: .transforms.ray-test
+
+# Use default rule inherited from makefile.common
+image:: .transforms.ray-image
+
+# Use default rule inherited from makefile.common
+venv:: .transforms.ray-venv
+
+test-src:: .transforms.test-src
+
+test-image:: .transforms.ray-test-image
+
+build:: build-dist image
+
+publish:: publish-dist publish-image
+
+publish-image:: .transforms.publish-image-ray
+
+setup:: .transforms.setup
+
+# distribution versions is the same as image version.
+set-versions:
+	$(MAKE) TRANSFORM_PYTHON_VERSION=${CODE_QUALITY_PYTHON_VERSION} TOML_VERSION=$(DOCKER_IMAGE_VERSION) .transforms.set-versions
+        
+build-dist:: set-versions .defaults.build-dist 
+
+publish-dist:: .defaults.publish-dist
+
+run-cli-sample:.
+	$(MAKE) RUN_FILE=$(TRANSFORM_NAME)_transform_ray.py \
+                RUN_ARGS="--run_locally True --data_local_config \"{ 'input_folder' : '../test-data/input', 'output_folder' : '../output'}\"  \
```

Comment:
Does this run w/o a cli error on --data_local_config?  I would expect an error w/o the change I suggested above on DAF.add_input_params(parser).